package-lock.json

```diff
@@ -21821,7 +21821,7 @@
     "@types/json5": {
       "version": "0.0.29",
       "resolved": "https://registry.npmjs.org/@types/json5/-/json5-0.0.29.tgz",
-      "integrity": "sha1-7ihweulOEdK4J7y+UnC86n8+ce4=",
+      "integrity": "sha512-dRLjCWHYg4oaA77cxO64oO+7JwCwnIzkZPdrrC71jQmQtlhM556pwKo5bUzqvZndkVbeFLIIi+9TC40JNF5hNQ==",
       "dev": true
```

Comment:
There are lots of changes on this file which I can not verify as I also get different lock file on main branch as well.

Reply:
yeah! and if I didn't push this all the GH actions were failing.